mlkem/mlkem_native_config.h

```diff
@@ -443,6 +443,65 @@
    }
 */
 
+/******************************************************************************
+ * Name:        MLK_CONFIG_CUSTOM_ALLOC_FREE [EXPERIMENTAL]
+ *
+ * Description: Set this option and define `MLK_CUSTOM_ALLOC` and
+ *              `MLK_CUSTOM_FREE` if you want to use custom allocation for
+ *              large local structures or buffers.
+ *
+ *              By default, all buffers/structures are allocated on the stack.
+ *              If this option is set, most of them will be allocated via
+ *              MLK_CUSTOM_ALLOC.
+ *
+ *              Parameters to MLK_CUSTOM_ALLOC:
+ *              - T* v: Target pointer to modify. Can be assumed to be NULL.
+ *              - T: Type of structure to be allocated
+ *              - N: Number of elements to be allocated.
+ *
+ *              Parameters to MLK_CUSTOM_FREE:
+ *              - T* v: Target pointer to free. May be NULL.
+ *              - T: Type of structure to be freed.
+ *              - N: Number of elements to be freed.
```

Comment:
Out of interest: When would you ever need the T and N?

Reply:
I can't follow. This question is about free, not alloc.

Reply:
Ah. I can't name an example off-hand, but it seems cleaner to provide the same information as for alloc. I initially had it to be able to include the zeroization.